src/components/LeftSidebar/ConversationsList/ConversationsList.vue

```diff
@@ -1,145 +0,0 @@
-  - published by the Free Software Foundation, either version 3 of the
-  - License, or (at your option) any later version.
-  -
-  - This program is distributed in the hope that it will be useful,
-  - but WITHOUT ANY WARRANTY; without even the implied warranty of
-  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-  - GNU Affero General Public License for more details.
-  -
-  - You should have received a copy of the GNU Affero General Public License
-  - along with this program. If not, see <http://www.gnu.org/licenses/>.
--->
-
-<template>
-	<ul class="conversations">
-		<Conversation v-for="item of conversationsList"
-			:key="item.id"
-			:item="item"
-			@click="handleConversationClick(item)" />
-		<template v-if="!initialisedConversations">
-			<LoadingPlaceholder type="conversations" />
-		</template>
-		<Hint v-else-if="searchText && !conversationsList.length"
-			:hint="t('spreed', 'No matches')" />
-	</ul>
-</template>
-
-<script>
-import { emit } from '@nextcloud/event-bus'
-
-import isMobile from '@nextcloud/vue/dist/Mixins/isMobile.js'
-
-import Hint from '../../Hint.vue'
-import LoadingPlaceholder from '../../LoadingPlaceholder.vue'
-import Conversation from './Conversation.vue'
-
-import { EventBus } from '../../../services/EventBus.js'
-
-export default {
-	name: 'ConversationsList',
-	components: {
-		Conversation,
-		Hint,
-		LoadingPlaceholder,
-	},
-	mixins: [isMobile],
-	props: {
-		searchText: {
-			type: String,
-			default: '',
-		},
-
-		conversationsList: {
-			type: Array,
-			required: true,
-		},
-
-		initialisedConversations: {
-			type: Boolean,
-			default: true,
-		},
-	},
-
-	data() {
-		return {
-			isFetchingConversations: false,
-		}
-	},
-
-	mounted() {
-		EventBus.$on('route-change', this.onRouteChange)
-		EventBus.$once('joined-conversation', ({ token }) => {
-			this.scrollToConversation(token)
-		})
-	},
-
-	beforeDestroy() {
-		EventBus.$off('route-change', this.onRouteChange)
-	},
-
-	methods: {
-		scrollToConversation(token) {
-			// FIXME: not sure why we can't scroll earlier even when the element exists already
-			// when too early, Firefox only scrolls a few pixels towards the element but
-			// not enough to make it visible
-			setTimeout(() => {
-				const conversation = document.getElementById(`conversation_${token}`)
-				if (!conversation) {
-					return
-				}
-				this.$nextTick(() => {
-					conversation.scrollIntoView({
-						behavior: 'smooth',
-						block: 'start',
-						inline: 'nearest',
-					})
-				})
-			}, 500)
-		},
-		onRouteChange({ from, to }) {
-			if (from.name === 'conversation'
-				&& to.name === 'conversation'
-				&& from.params.token === to.params.token) {
-				// this is triggered when the hash in the URL changes
-				return
-			}
-			if (from.name === 'conversation') {
-				this.$store.dispatch('leaveConversation', { token: from.params.token })
-				if (to.name !== 'conversation') {
-					this.$store.dispatch('updateToken', '')
-				}
```

Comment:
Need to be added after #9058 approval

Reply:
Nice catch!

#9058 is a backport, so it should be fixed anyway.

Reply:
Sadly, but no. It's a copied method from deleted file